docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,83 @@
+# Relative layout
+
+The `RelativeLayout` is a layout class, in which each side of children can be specified as relative to sibling view or parent. The `Target` and the `RelativeOffset` properties are used to define a relationship with each other. The following figure shows the relative layout position rectangles with `Target` and `RelativeOffset`. The arrows in the preview indicate where the `LeftTarget` and the `RightTarget` is pointing:
```

Comment:
There was a slight grammatical issue, and to make sentence active:
```suggestion
The `RelativeLayout` is a layout class in which each side of children can be specified as relative to a sibling view or parent. The `Target` and the `RelativeOffset` properties define a relationship with each other. 

The following figure shows the relative layout position of rectangles with `Target` and `RelativeOffset`. The arrows in the preview indicate where the `LeftTarget` and the `RightTarget` is pointing:

Reply:
Done.